### ISSUE TITLE
Installation failure because of peer dependencies.

### ISSUE THREAD

**Issue by bunchesofdonald** [apiaryio/dredd#905]
I'm attempting to install Dredd but I am getting a `EPEERINVALID` error when installing with `npm install -g dredd`:

```
/usr/bin/dredd -> /usr/lib/node_modules/dredd/bin/dredd
npm ERR! Linux 4.9.49-moby
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "dredd"
npm ERR! node v4.8.5
npm ERR! npm  v2.15.11
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package fury@3.0.0-beta.5 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer fury-adapter-apib-parser@0.9.0 wants fury@3.0.0-beta.4
npm ERR! peerinvalid Peer fury-adapter-swagger@0.13.4 wants fury@3.0.0-beta.4

npm ERR! Please include the following file with any support request:
npm ERR!     /groot/npm-debug.log```

**Comment by honzajavorek** [comment]
This is interesting. With newer `npm` versions it seems this is only issued as a warning, not as an error.

**Comment by honzajavorek** [comment]
This is going to be solved with the next upgrade of [Dredd Transactions](https://github.com/apiaryio/dredd-transactions) in Dredd. There are some changes we want to add before upgrading, so although in Dredd Transactions this is fixed, it still may take some time before it gets to Dredd.

**Comment by honzajavorek** [comment]
@bunchesofdonald This should be now fixed in [v4.7.1](https://github.com/apiaryio/dredd/releases/tag/v4.7.1). Would you mind to confirm it's okay now?
